reference/src/main/java/oshi/api/NoSuchQueryMethodException.java

```diff
@@ -0,0 +1,5 @@
+package oshi.api;
+
+public class NoSuchQueryMethodException extends RuntimeException {
```

Comment:
Actually this probably needs to be changed, but I'm not sure what to do in this case. This exception is thrown if the user tries to query an attribute that has no query method configured. This can happen in two ways:

- We forgot to implement the query method or annotate it with `@Query(ATTRIBUTE_NAME)`.
- The user tried to query an attribute that requires an external software package that wasn't found or the privilege requirements are not met

Reply:
In the first case, it's a developer thing and a Runtime Excption is ok.  In the second, I don't want to choke because the user didn't install an external software package.  A pleasantly polite log message suggesting they do so would be preferred.

Reply:
In both cases, the attribute's value in the container will remain `null`. This is why I have all of the attribute data types set to be boxed types, but they should eventually be `Optional`s instead. We can decide later if we want to include the error message in the `Optional` by building an `OshiResult` class.